crates/fuel-core/src/graphql_api/api_service.rs

```diff
@@ -278,16 +275,22 @@ where
         .extension(ViewExtension::new())
         .finish();
 
+    let graphql_endpoint = "/v1/graphql";
+    let graphql_subscription_endpoint = "/v1/graphql-sub";
+
+    let graphql_playground =
+        || render_graphql_playground(graphql_endpoint, graphql_subscription_endpoint);
+
     let router = Router::new()
         .route("/v1/playground", get(graphql_playground))
```

Comment:
We can support old endpoint under v1 and new under v2=)

Reply:
What's the point ?

Reply:
the old playground has security issues too so I'm not sure we should keep it

Reply:
For me when you perform an update, you can except some UI changes as soon as it doesn't break any code I don't think anyone is doing scrapping on the graphql playground and I see no point of using the previous ones all features seems to exist on this one too.

Reply:
However I don't want to block anything so it's a deal breaker for you @xgreenx I'm ok to walk beside you on that.